libs/application/template-api-modules/src/lib/modules/templates/accident-notification/types/index.ts

```diff
@@ -0,0 +1,9 @@
+export interface FileAttachment {
+  name: string
+  url: string
+}
+
+export interface NodemailAttachment {
+  filename: string
+  href: string
+}
```

Comment:
Does nodemailer include a type definition for attachments that we can use?

Would be best to skip these interfaces and prepare the attachments once using the `nodemailer` attachment type instead of mapping the attachments twice with different types used.